be/src/exec/vectorized/schema_scanner/schema_task_runs_scanner.cpp

```diff
@@ -139,9 +140,29 @@ Status SchemaTaskRunsScanner::fill_chunk(ChunkPtr* chunk) {
             break;
         }
         case 8: {
-            // ERROR_CODE
+            // EXPIRE_TIME
```

Comment:
Will changing this order affect compatibility during upgrades?

Reply:
Task Framework is 2.3 feature. So the target doesn't need to consider compatibility yet.